src/pages/profile/edit.jsx

```diff
@@ -5,102 +5,128 @@ import { Container, Grid } from '../../components/layout/index';
 import { Heading, Button } from '../../components/UI';
 
 const EditProfile = () => {
+  // TODO: get role, depending on role, different info will be shown (getNotifInfo). Text is not final!
+  const getNotifInfo = (role) => {
+    switch (role) {
+      case 'user':
+        return 'Bepaal hoe je meldingen krijgt wanneer je aanvraag om deel te nemen aan een leerkans gereviewed werd.';
+      case 'admin':
+        return 'Bepaal hoe je meldingen krijgt wanneer organisaties een nieuwe leerkans aanmaken.';
+      case 'institution':
+        return 'Bepaal hoe je meldingen krijgt wanneer jouw leerkans werd goedgekeurd en wanneer studenten zich aanmelden aan een leerkans. ';
+      default:
+        return 'Bepaal hoe je meldingen krijgt.';
+    }
+  };
+
+  const editProfile = (values) => {
+    // TODO: make editting profile possible
+    console.log(values);
+  };
+
+  const editNotifications = (values) => {
+    // TODO: make editting notifications possible
+    console.log(values);
+  };
+
+  // TODO: get profile info from database (if logged in, if not redirect, possibly through React router)
   const profile = {
     email: 'john.doe@gmail.com',
     firstName: 'John',
```

Comment:
First and last name have been merged into a single field `name`.

Unless we assume that the first space is the delimiter (which may not be accurate for some users), there is no way to know which is which.